### PR TITLE
updated form on profile page to match the one on the registration page

### PR DESCRIPTION
**Pull request by apluscs**
1. profile-1
2. profile-2

### CODE REVIEW

File: src/main/webapp/profile.html [hunk 1]
```diff
@@ -14,8 +14,9 @@
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
       integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
     <script src="profile.js"></script>
+    <script src="register.js"></script>    
   </head>
-  <body>
+  <body onload="onloadProfile()">
```
Comment (meagancheese) [inline src/main/webapp/profile.html]: In another branch I was doing onProfileLoad()

File: src/main/webapp/profile.html [hunk 2]
```diff
@@ -39,36 +40,48 @@
...
-          <label for="user-job" class="col-sm-2 col-form-label">Job</label>
-          <div class="col-sm-10">
-            <input type="text" readonly class="form-control-plaintext editable" id="user-job" value="Software Engineer">
+        <div class="form-row">
+          <div class="col-md-6 mb-3">
+            <label for="company-field">Company</label>
+            <input type="text" class="form-control-plaintext editable" id="company-field" value="Google">
+          </div>
+          <div class="form-group col-md-6 mb-3">
+            <label for="job-field">Job</label>
```
Comment (meagancheese) [inline src/main/webapp/profile.html:67]: Why is this showing up as a dropdown even when it's set?

File: src/main/webapp/profile.js [hunk 3]
```diff
@@ -12,6 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function onloadProfile() {
```
Comment (meagancheese) [inline src/main/webapp/profile.js]: Again, switch to onProfileLoad()

**Review by meagancheese — CHANGES_REQUESTED** [review]
Small changes, one for conformity, the other for aesthetics.

File: src/main/webapp/profile.html (hunk 2 above)
Comment (apluscs) [reply src/main/webapp/profile.html:67]: Software Engineer is the first option

Comment (meagancheese) [reply src/main/webapp/profile.html:67]: When we are showing the user the current info they inputted, we shouldn't have the job entry look like it's still modifiable.

**Review by meagancheese — CHANGES_REQUESTED** [review]
See comment about Job display.

File: src/main/webapp/profile.html (hunk 2 above)
Comment (apluscs) [reply src/main/webapp/profile.html:67]: Updated screenshots.

**Review by meagancheese — APPROVED** [review]
LGTM

File: src/main/webapp/profile.html [hunk 4]
```diff
@@ -39,36 +40,49 @@
...
-          <label for="user-company" class="col-sm-2 col-form-label">Company</label>
-          <div class="col-sm-10">
-            <input type="text" readonly class="form-control-plaintext editable" id="user-company" value="Google">
+        <div class="form-row">
+          <div class="col-md-6 mb-3">
+            <label for="first-name-field">First name</label>
+            <input type="text" readonly class="form-control-plaintext editable" id="first-name-field" placeholder="First name" value="Tess" required>
+          </div>
+          <div class="col-md-6 mb-3">
+            <label for="first-name-field">Last name</label>
```
Comment (jalexanderqed) [inline src/main/webapp/profile.html]: It seems like this should be `for="last-name-field"`

**Review by jalexanderqed — APPROVED** [review]
One small fix requested. Approving now.

**Pull request merged**